app/src/main/java/org/p2p/wallet/history/ui/token/TokenHistoryFragment.kt

```diff
@@ -177,7 +185,7 @@ class TokenHistoryFragment :
     }
 
     override fun hideBuyActionButton() {
-        binding.viewActionButtons.setActionButtonVisible(ActionButton.BUY_BUTTON, isVisible = false)
+        binding.viewActionButtons.setActionButtonIsVisible(ActionButton.BUY_BUTTON, isVisible = false)
```

Comment:
setActionButtonVisibility

Reply:
there are three Visibilities: VISIBLE, INVISIBLE, GONE. Which one will be used if i pass true or false?